packages/ckeditor5-typing/src/texttransformation.js

```diff
@@ -249,8 +248,9 @@ function normalizeTransformations( config ) {
 	const configured = config.include.concat( extra ).filter( isNotRemoved );
 
 	return expandGroupsAndRemoveDuplicates( configured )
-		.filter( isNotRemoved ) // Filter out 'remove' transformations as they might be set in group
+		.filter( isNotRemoved ) // Filter out 'remove' transformations as they might be set in group.
 		.map( transformation => TRANSFORMATIONS[ transformation ] || transformation )
+		.filter( transformation => typeof transformation === 'object' ) // Filter out transformations set as string that has not been found.
```

Comment:
An existing test started to fail after the batch changes has been introduced. This was a fix that was needed. It seems that previously, the test was a false positive.